index.html

```diff
@@ -1,71 +1,268 @@
-                            <li>
-                                <a href="#">
-                                    <img src="images/twitter.svg" alt="twitter">
-                                </a>
-                            </li>
-                            <li>
-                                <a href="">
-                                    <img src="images/Linkedin.svg" alt="Linkedin">
-                                </a>
-                            </li>
-                            <li>
-                                <a href="">
-                                    <img src="images/microverse.svg" alt="Meta">
-                                </a>
-                            </li>
-                            <li>
-                                <a href="">
-                                    <img src="images/github.svg" alt="github">
-                                </a>
-                            </li>
-                            <li>
-                                <a href="">
-                                    <img src="images/victory.svg" alt="victory">
-                                </a>
-                            </li>
-                        </ul>
-                </div>
-                
-            </div>
-        </section>
-       <!--End of Section-->
+    <!--Starting  of work  section-->
+    <section class="work-section">
+      <!--First Card starting-->
+      <div class="card">
+        <img src="images/Snapshoot Portfolio.svg" alt="Project Screenshot" />
+        <div class="card-text">
+          <h2>Tonic</h2>
+          <p>
+            <strong>CANOPY</strong>
+            <i class="fa-solid fa-circle fa-2xs"></i> Back End Dev
+            <i class="fa-solid fa-circle fa-2xs"></i> 2015
+          </p>
+          <p class="detail">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul>
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+          </ul>
+          <div class="project-details-btn">
+            <input type="button" value="See Project" />
+          </div>
+        </div>
+      </div>
+     <!--Second Card starting-->
+      <div class="card">
+        <img src="images/Multi-post-stories.svg" alt="Project Screenshot" />
+        <div class="card-text">
+          <h2>Multi-Post Stories</h2>
+          <p>
+            <strong>CANOPY</strong>
+            <i class="fa-solid fa-circle fa-2xs"></i> Back End Dev
+            <i class="fa-solid fa-circle fa-2xs"></i> 2015
+          </p>
+          <p class="detail">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul>
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+          </ul>
+          <div class="project-details-btn">
+            <input type="button" value="See Project" />
```

Comment:
- [ ] please kindly add and follow the button interaction and styling: Hover, pressed .. Like in the original design. Kindly do this for all buttons. Nice work.

![Screenshot (1104)](https://github.com/anshuman7negi/my-portfolio/assets/65298048/ec7ca230-96c4-4f9d-9718-8aa8365c24b5)